### ISSUE TITLE
Extract dates from LHOM index entries

### ISSUE THREAD

**Issue by nathangibson** [usaybia/usaybia-data#137]
@hannafriedel You can take a look at the **Dates** tab in this spreadsheet https://docs.google.com/spreadsheets/d/1ujiT91ua3sA-WX86OWpuE-gDD_E-zONpI1dP70pXdWw/edit#gid=216912508

The Google Sheets functions that are relevant are RegExMatch, RegExExtract, and RegExReplace, all of which use RegEx. You can find out more about them [here](https://support.google.com/docs/table/25273) or look at the examples in the columns I've already done. 

I've put the regular expressions in rows 2-4 so that you can easily edit them without having to change the formula each time you do. So far I've only attempted the death dates, and they're not perfect yet, so you'll need to enter the regex for the other columns.

If you need more columns let me know. 

Here's the process: 

1. Edit the RegEx for Column C to correctly extract the date-related text from the LHOM Entry in Column B. It is mostly doing this already but there are some errors. 
2. Edit the RegEx for columns E-G to correctly extract death dates. The current RegEx needs some adjustment. Also it doesn't yet catch deaths from date ranges like 557–629 (where 629 is the death date). 
3. Add RegEx for Birth, Active, Reign (uses \`reg\`). Centuries will be interesting ... I've put columns for Active century AH and Active century AD but you might need something more. 
4. Please get it as good as you can using RegEx, then have me check. There will definitely be things a formula can't capture correctly. Once we have it as good as we can, I'll ask you to go through and check and correct what is there, but only after we're done adjusting the RegEx. 

You can ask me questions here or if it relates to a particular cell in the spreadsheet you could tag me in a comment there. 

Make sense?

**Comment by hannafriedel** [comment]
I corrected all dates.
